### DISCUSSION TITLE
Zigbee2mqtt devices loosing

### DISCUSSION THREAD

**Discussion by msteingr** [Koenkk/zigbee2mqtt#19206] [General]
Hello not sure if it should be posted here but it was thinking it could be 
 I have about 150 zigbee  devices being about 70 lamps/router and  the others battery end devices connected to my HAS network  
 I loose each day abou 10 of my end devices  which I have to reconnect each time..

 Is there a way to make it more stable
 I use a conBee II stick ..  and raspberry 4 

thanks your suggestions

marc

**Comment by lux73** [comment]
First please pay your Post more Attention: **Zigbvee2mqtt, end divides, congee II stick** !?!

this seems you don't wan't spend 2 Minutes for your Post - so nobody will spend there spare time to help **you**

back to your Problem: 150 Zigbee Devices is alot for only one (1) Coordinator - i don't know about your ConBee Stick but i would assume there is a "Wall" and your Stick could not handle this amount of Devices

other Zigbee Stick available could even not handle this amount of Devices - it is recommended to split your Zigbee Network in two different Setups, so any Stick have to handle only 75 Devices. This works like a charme

if your HAS is able to split your Zigbee Network and handle two Coordinator? I don't know - with bare z2m Installation this is possible ;-)

**Reply by msteingr** [reply]
Thanks for your time I corrected my misspelling, when you say split do you mean two raspberry and independent systems ?  Or should this work in on  application?

thanks again
Marc

**Comment by Nardol** [comment]
> back to your Problem: 150 Zigbee Devices is alot for only one (1) Coordinator - i don't know about your ConBee Stick but i would assume there is a "Wall" and your Stick could not handle this amount of Devices


If there are routers I don't see why it should be better by separating into two distinct Zigbee networks, [see FAQ](https://www.zigbee2mqtt.io/guide/faq/#i-read-that-zigbee2mqtt-has-a-limit-of-20-devices-when-using-a-cc2530-cc2531-adapter-is-this-true).

But what is sure is the Combee is not the best recommendation for a good Zigbee network.
You should read [supported adapters](https://www.zigbee2mqtt.io/guide/adapters/) for more information.

Have you also read [instructions to improve Zigbee](https://www.zigbee2mqtt.io/advanced/zigbee/02_improve_network_range_and_stability.html)?
